web/src/stream_ui_updates.ts

```diff
@@ -36,6 +36,22 @@ function settings_button_for_sub(sub: StreamSubscription): JQuery {
     );
 }
 
+function settings_button_for_archive(sub: StreamSubscription): JQuery {
```

Comment:
We can skip having this functions since we do not use them more than once as of now.